src/Orleans.Runtime/Orleans.Runtime.csproj

```diff
@@ -12,15 +12,17 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsOptionsVersion)" />
+    <Compile Remove="Providers\**" />
+    <EmbeddedResource Remove="Providers\**" />
+    <None Remove="Providers\**" />
```

Comment:
Are these elements needed? I think we can we just remove them since the folder is empty.

Reply:
I'll fix it in a subsequent PR